index.html

```diff
@@ -324,11 +351,35 @@
     .half-margin-bottom {
         margin-bottom: 50px;
     }
-    .name-text-container {
-        /*width: 700px;*/
-        /*padding: 0 100px;*/
-    }
 
+    .padding-container > img {
+        height: 100%;
+        width: 100%;
+    }
+    .padding-container {
+        padding: 25px;
+    }
+    .tech-container {
+        min-width: 30%;
+        height: 150px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    .tech-container .tech {
+        max-width: 250px;
+    }
+    .aws-logo > .tech {
+        max-width: 150px;
+    }
+    .tech-body {
+        width:100%;
```

Comment:
100% width plus margins hangs outside of the viewport. This should keep that from happening.
```suggestion
        width:calc(100% - 100px);
```

Reply:
Good catch. I'll just remove the width: 100%. It was unnecessary

Reply:
Jk I'll use calc